src/main/java/nl/b3p/tailormap/api/persistence/OIDCConfiguration.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2023 B3Partners B.V.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+package nl.b3p.tailormap.api.persistence;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Version;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class OIDCConfiguration {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @Version private Long version;
+
+  @NotNull private String name;
```

Comment:
Not technically necessary, but adding a logo image would be helpful for users. We can reuse code for the image of an application.